app/src/androidTest/java/com/duckduckgo/app/di/FakeJobSchedulerModule.kt

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2018 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.app.di
+
+import android.app.job.JobInfo
+import android.app.job.JobScheduler
+import android.app.job.JobWorkItem
+import dagger.Module
+import dagger.Provides
+import javax.inject.Singleton
+
+@Module
+class FakeJobSchedulerModule {
```

Comment:
How would you feel about calling this `StubJobSchedulerModule`? I feel like there isn't a clear distinction between Stubs and Fakes and there is little benefit in separating them so  it makes sense to use `Stub` for both. It seems I'm not alone https://stackoverflow.com/questions/346372/whats-the-difference-between-faking-mocking-and-stubbing#346440.